### ISSUE TITLE
Plugin attempting to clear CF Ent Cache when Sucuri is selected

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#115]
```
delightfulmomfood.com
NerdPress_Cloudflare_Client::handlePostCacheTransition publish publish
"{"success":false,"errors":[{"code":9106,"message":"Missing X-Auth-Key, X-Auth-Email or Authorization headers"}]}"
CF-Ray: "5a90e4d69943ec8e-DFW"
URL: https://delightfulmomfood.com/daily-agenda-book-with-free-printables/
Cleared: full
```

Site had Sucuri Firewall selected (correctly)...but was trying to clear the Cloudflare Enterprise cache anyway.

**Comment by blogtutor** [comment]
Just confirmed on dev.blogtutor.com.  Plugin will attempt to call Cloudflare API even if it's set to "Sucuri" or "None."  We need to add a check to make sure it only tries to clear CF if the Firewall Selection is Cloudflare Enterprise.

**Comment by blogtutor** [comment]
Closed with 0.7.2 release.
